lab_5_scrapper/scrapper.py

```diff
@@ -160,6 +283,10 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
             article_id (int): Article id
             config (Config): Configuration
         """
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
```

Comment:
is it stated in docs that Article instance should be an attribute?

Reply:
Yes
"Each instance should also have an additional self.article attribute, initialized with a new instance of stubs.core_utils.ctlr.article.article.Article class."

Reply:
thank you.